rpm/qtmultimedia.spec

```diff
@@ -187,67 +186,67 @@ rm -rf %{buildroot}/%{_includedir}/qt5/Qt
 %defattr(-,root,root,-)
 %license LICENSE.LGPLv* LGPL_EXCEPTION.txt
 %license LICENSE.GPLv3 LICENSE.FDL
-%{_libdir}/libQt5Multimedia.so.5
-%{_libdir}/libQt5Multimedia.so.5.*
-%{_libdir}/libQt5MultimediaQuick_p.so.5
-%{_libdir}/libQt5MultimediaQuick_p.so.5.*
+%{_qt5_libdir}/libQt5Multimedia.so.5
+%{_qt5_libdir}/libQt5Multimedia.so.5.*
+%{_qt5_libdir}/libQt5MultimediaQuick_p.so.5
+%{_qt5_libdir}/libQt5MultimediaQuick_p.so.5.*
 
 %files devel
 %defattr(-,root,root,-)
-%{_libdir}/libQt5Multimedia.so
-%{_libdir}/libQt5MultimediaQuick_p.so
-%{_libdir}/libqgsttools_p.so
-%{_libdir}/libQt5Multimedia.prl
-%{_libdir}/libQt5MultimediaQuick_p.prl
-%{_libdir}/libqgsttools_p.prl
-%{_libdir}/pkgconfig/*
-%{_includedir}/qt5/*
-%{_datadir}/qt5/mkspecs/
-%{_libdir}/cmake/
+%{_qt5_libdir}/libQt5Multimedia.so
+%{_qt5_libdir}/libQt5MultimediaQuick_p.so
+%{_qt5_libdir}/libqgsttools_p.so
+%{_qt5_libdir}/libQt5Multimedia.prl
+%{_qt5_libdir}/libQt5MultimediaQuick_p.prl
+%{_qt5_libdir}/libqgsttools_p.prl
+%{_qt5_libdir}/pkgconfig/*
```

Comment:
I'd say let's not use this extra level of indirection when qt, among everything else, really gets installed to the common libdir. Overriding this directory would be funky.

Furthermore here it would be even sort of wrong since pkgconfig directory is defined by pkgconfig and not qt.